dist/rest/api-request.js

```diff
@@ -0,0 +1,34 @@
+"use strict";
```

Comment:
How is it related to this PR? I think you added it by mistake

Reply:
it's dist @tgorskievo

Reply:
It means that something incorrectly built, so I wonder if the actual master works properly.